safehttp/plugins/xsrf/xsrf.go

```diff
@@ -66,9 +96,12 @@ func (p *Interceptor) Before(w safehttp.ResponseWriter, r *safehttp.IncomingRequ
 		return w.ClientError(safehttp.StatusUnauthorized)
 	}
 
-	if ok := xsrftoken.Valid(tok, p.AppKey, userID, r.Host()+r.Path()); !ok {
+	if ok := xsrftoken.Valid(tok, i.AppKey, userID, r.URL.String()); !ok {
```

Comment:
What prompted this change?

Reply:
The `actionID` should indicate the action the user is taking and having the URL seemed better than doing the string concatenation twice as before. Alternatively, we can also add the HTTP method used.

Reply:
The URL is not just host + path, so you need to be careful here. It's also query params and and fragments, right?

Reply:
Oh right, that could introduce problems, I will follow the example `xsrftoken` gives the `Generate` godoc (HTTP method and path). WDYT?

Reply:
I'd keep the host too.

@empijei WDYT?

Reply:
@mihalimara22 Please file an issue about this and resolve. I can merge then. :)